### PR TITLE
Updated UG for Tabview, RadioButton and CheckBox

### PR DESCRIPTION
**Pull request by KarthikaRavichandranSF4617**
## Description ##

- Updated UG for Tabview, RadioButton and CheckBox

### CODE REVIEW

File: MAUI/CheckBox/Visual-Customization.md [hunk 1]
```diff
@@ -301,4 +301,26 @@ SfCheckBox checkBox = new SfCheckBox
...
+
+<syncfusion:SfCheckBox Text="CheckBox" ContentSpacing="25"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfCheckBox checkBox = new SfCheckBox
+{
+    Text = "CheckBox",
+    ContentSpacing="25"
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Visual-Customization.md:320]: Is this correct? did you run it?

File: MAUI/Radio-Button/Visual-Customization.md [hunk 2]
```diff
@@ -217,4 +217,26 @@ SfRadioButton radioButton = new SfRadioButton
 };
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Content spacing
+This feature allows users to adjust the content spacing in the [`SfRadioButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfRadioButton.html) control.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Radio-Button/Visual-Customization.md:223]: Modify the content based on the existing features

File: MAUI/TabView/Tab-Item-Customization.md [hunk 3]
```diff
@@ -538,3 +538,28 @@ tabView.ContentTransitionDuration = 300;
 {% endhighlight %}
 
 {% endtabs %}
+
+## Image size
+
+You can customize the image size in the .NET MAUI TabView control by setting the ImageSize property.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:544]: You can customize the image size in the .NET MAUI TabView control by setting the `ImageSize` property.

File: MAUI/TabView/Tab-Item-Customization.md [hunk 4]
```diff
@@ -538,3 +538,28 @@ tabView.ContentTransitionDuration = 300;
...
+</tabView:SfTabView>
+{% endhighlight %}
+
+{% highlight C# %}
+var tabView = new SfTabView();
+var tabItems = new TabItemCollection
+{
+    new SfTabItem()
+    {
+        ImageSize = "20",
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/TabView/Tab-Item-Customization.md:560]: Please check and add the code properly.

File: MAUI/CheckBox/Visual-Customization.md [hunk 5]
```diff
@@ -301,4 +301,26 @@ SfCheckBox checkBox = new SfCheckBox
 };
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Content spacing
+This feature allows users to adjust the content spacing in the [`SfCheckBox`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/CheckBox/Visual-Customization.md:307]: The `ContentSpacing` property is used to adjust the content spacing in the [`SfCheckBox`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html) control.

Comment (KarthikaRavichandranSF4617) [reply MAUI/CheckBox/Visual-Customization.md]: Updated the content.

File: MAUI/CheckBox/Visual-Customization.md (hunk 1 above)
Comment (KarthikaRavichandranSF4617) [reply MAUI/CheckBox/Visual-Customization.md]: Updated the code changes.

File: MAUI/TabView/Tab-Item-Customization.md (hunk 3 above)
Comment (KarthikaRavichandranSF4617) [reply MAUI/TabView/Tab-Item-Customization.md]: Updated the content.

File: MAUI/TabView/Tab-Item-Customization.md (hunk 4 above)
Comment (KarthikaRavichandranSF4617) [reply MAUI/TabView/Tab-Item-Customization.md]: Updated the code changes.

**Review by naveenkumar-sanjeevirayan — APPROVED** [review]

**Pull request merged**